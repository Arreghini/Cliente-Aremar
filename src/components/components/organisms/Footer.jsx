import React from 'react'

function Footer() {
  return (
    <footer className="relative z-10 text-center p-4 font-body text-neutral-oscuro bg-neutral-claro bg-opacity-70">
        Â© 2025 Edificio Aremar. Todos los derechos reservados.
      </footer>
  )
}

export default Footer