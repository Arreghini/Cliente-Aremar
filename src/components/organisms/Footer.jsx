import React from 'react'

function Footer() {
  return (
    <footer className="relative z-10 text-center font-body text-neutral-oscuro bg-neutral-claro bg-opacity-70">
  <div className="max-w-7xl mx-auto p-4">
    <p className="mb-2">
      Edificio Aremar - Av. Calle 1 N°1919, Las Toninas, Argentina
    </p>
    <p>© 2025 Aremar. Todos los derechos reservados.</p>
  </div>
</footer>
  )
}

export default Footer;
