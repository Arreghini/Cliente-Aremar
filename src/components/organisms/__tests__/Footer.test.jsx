import { describe, expect, test } from 'vitest';
import { render, screen } from '@testing-library/react';
import Footer from '../Footer';

describe('Footer', () => {
  test('se renderiza sin errores', () => {
    render(<Footer />);
    const footer = screen.getByRole('contentinfo');
    expect(footer).toBeInTheDocument();
  });

  test('contiene la dirección del edificio', () => {
    render(<Footer />);
    expect(
      screen.getByText('Edificio Aremar - Av. Calle 1 N°1919, Las Toninas, Argentina')
    ).toBeInTheDocument();
  });

  test('contiene el aviso de derechos reservados', () => {
    render(<Footer />);
    expect(screen.getByText('© 2025 Aremar. Todos los derechos reservados.')).toBeInTheDocument();
  });

  test('tiene clases de Tailwind correctas en el footer', () => {
    render(<Footer />);
    const footer = screen.getByRole('contentinfo');
    expect(footer).toHaveClass('relative z-10 text-center font-body text-neutral-oscuro bg-neutral-claro bg-opacity-70');
  });

  test('el contenedor interno tiene clases de Tailwind correctas', () => {
    render(<Footer />);
    const innerDiv = screen.getByText(/Edificio Aremar/).parentElement;
    expect(innerDiv).toHaveClass('max-w-7xl mx-auto p-4');
  });

  test('snapshot del Footer', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
