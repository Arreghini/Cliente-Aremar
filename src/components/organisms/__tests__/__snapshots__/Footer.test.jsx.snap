// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > snapshot del Footer 1`] = `
<div>
  <footer
    class="relative z-10 text-center font-body text-neutral-oscuro bg-neutral-claro bg-opacity-70"
  >
    <div
      class="max-w-7xl mx-auto p-4"
    >
      <p
        class="mb-2"
      >
        Edificio Aremar - Av. Calle 1 N°1919, Las Toninas, Argentina
      </p>
      <p>
        © 2025 Aremar. Todos los derechos reservados.
      </p>
    </div>
  </footer>
</div>
`;
